src/migrations/project/r-migration-commands/m144_145_add_volume_type_default_value_to_collider_box_element.ts

```diff
@@ -0,0 +1,26 @@
+import { r, RecordNode, rtp, RT, sn, en } from "../../../r";
+import { IOrder } from "../../IOrder";
+
+
+/**
+ * Fixing co_type for Actions migrated in earlier migration
+ */
+class Migration implements IOrder {
+  execute(projectJson: unknown) {
+    const pJson = projectJson as RecordNode<RT.project>;
+    const projectF = r.record(pJson);
+
+    const allColliderBoxElements = projectF.getAllDeepChildrenWithFilter(RT.element, el => el.props.element_type === "collider_box");
+
+    for (const el of allColliderBoxElements) {
+      const elementF = r.element(el);
+      elementF.set(rtp.element.element_type, en.ElementType.collider_volume);
+      elementF.set(rtp.element.volume_type, en.VolumeTypes.cube);
+    }
+
+    projectF.set(rtp.project.version, 145);
```

Comment:
Verify thé versioning from last couple of migrations. Easy to mess up versions

Reply:
Verified and checked